src/realm/realmReducers.js

```diff
@@ -38,6 +39,7 @@ const initialState = {
 
 const realmInit = (state: RealmState, action: RealmInitAction): RealmState => ({
   ...state,
+  canCreateStreams: action.data.can_create_streams,
```

Comment:
Not sure we want to be matching all these properties manually, or to keep them in the root of `realm` state. we likely want some object like 'attribs' and have all the plain ones.

Reply:
For now let's add them as we start using using them. (not much overhead)
If we add all of then now, then it will be just redundant info cached.